packages/theme/index.js

```diff
@@ -20,6 +77,18 @@ module.exports = {
 		console.log(chalk.cyan('	npm run lint'));
 		console.log('		Lints the source code.');
 		console.log();
+		console.log('We suggest that you begin by typing:');
+		if (path.resolve(process.cwd()) !== path.resolve(directory)) {
+			console.log(chalk.cyan('	cd ' + path.relative(process.cwd(), directory)));
+		}
+		console.log('	' + chalk.cyan('npm run transpile'));
+		console.log('	' + chalk.cyan('npm link build'));
+		console.log('This will build your theme library into ES5/CSS and then link that '
```

Comment:
Normally, for development internally, we link to the 'raw' version of the template for debugging.  Why are we suggesting using the transpiled one here?

Reply:
The ability to link "raw" templates is something solely affording to "@enact/*" libraries. We have special handling for `node_modules/@enact/*` to have webpack treat our code there like app-level code  (typescript/react/babel transpiling).  This is not the community standard and was done solely as an internal team "special case".  So for the case of 3rd party themes, yep, "transpile -> link `./build`" is the correct workflow.

Reply:
Aha, of course.